src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md

```diff
@@ -34,6 +36,18 @@ PDF Services APIs can connect with the Amazon S3 using pre-signed URLs for downl
 
 Please refer [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html) for more details.
 
+## Dropbox
+
+PDF Services APIs can connect with the Dropbox using pre-signed URLs for downloading and uploading the files to and from Dropbox location without the need for any credentials or access tokens.
```

Comment:
remove 'to and from'

Reply:
without the need for any credentials or access tokens. -> 'without the need for credentials or access token.'

Reply:
Done.